src/components/Content/Content.tsx

```diff
@@ -27,23 +27,14 @@ const styles = {
   },
 };
 
-const INITIAL_STATE = {
-  editing: false,
-  saving: false,
-  creating: false,
-  disabling: false,
-  confirmingDelete: false,
-  deleting: false,
-};
-
 const enhance = compose(provideThing, injectState, withRouter);
 
 class Content extends React.Component<any, any> {
-  state = INITIAL_STATE;
+  state = { contentState: '' };
```

Comment:
let's type this with an enum

Reply:
good idea!
what would be a good primary state? `displaying`?

Reply:
not sure
`initial`?